Framework/NexusGeometry/src/NexusGeometrySave.cpp

```diff
@@ -14,14 +12,20 @@
  * @date 07/08/2019
  */
 
-#include "MantidNexusGeometry/NexusGeometrySave.h"
+#include "MantidGeometry/Objects/IObject.h"
```

Comment:
This file seems to be introducing more features for the `saveInstrument` functionality, but the issue is about exposing the functionality "as-is" via an algorithm. Is there a good reason for making both sets of changes in one PR?

Reply:
artefacts from an older merge, ive checked out the NexusGeometrySave files from master now